validator/accounts/accounts_exit.go

```diff
@@ -180,7 +177,7 @@ func displayExitInfo(rawExitedKeys [][]byte, trimmedExitedKeys []string) {
 	}
 }
 
-func writeSignedVoluntaryExitJSON(ctx context.Context, sve *eth.SignedVoluntaryExit, outputDirectory string) error {
+func writeSignedVoluntaryExitJSON(_ context.Context, sve *eth.SignedVoluntaryExit, outputDirectory string) error {
```

Comment:
Could we not just delete the `ctx` completely?

Reply:
I could remove it i just noticed it wasn't used